cal/cal/static/js/app.js

```diff
@@ -111,29 +111,41 @@ analyticsApp.component('tags', {
 function TagsDetailCtrl($scope, $http) {
   var tagUrl = '/v1/tags/' + this.tagId + '/events';
   var eventweek = '/v1/tags/' + this.tagId + '/eventWeek';
+  var query_timezone = moment.tz.guess();
   $scope.tagDetails = [];
   $scope.tagEvents = [];
+  $scope.tagHours = this.tagHours;
 
   $http({method: 'GET', url: tagUrl + '.json' }).
   success(function successCallback(data) {
     for (var i = 0; i < data.results.length; i++) {
       var event = data.results[i]
       $scope.tagEvents.push({
-        start: event.start,
+        start: (new Date(event.start)).toString(),
```

Comment:
Does `toISOString` still work? If so, I think we should use `toISOString` because it's the less 'naive' version. In javascript datetimes at least, ISOString gives you a timezone aware string, so I'm wondering if that same principle can be applied here.